### DISCUSSION TITLE
Pro Case - Support radio silence

### DISCUSSION THREAD

**Discussion by timfee** [vercel/community#3101] [Help] [answered]
### Summary

Hi gang,

I’ve opened two related cases (because I couldn’t append to an in-progress case w/o an email reply) back on June 20.

I marked it as a top severity issue — there was read issues with my Vercel config store.

I haven’t heard a single thing from anyone yet about my issue. 

I know I’m only paying $20/month for a paid plan — but I do it gladly as I really want to support you — but it’s really a bad look to have no sense of urgency, follow-through or care for your paying customers. 

The issue was resolved when I re-built everything from scratch after not getting a reply.

I know I’m one small account, but this experience really leaves a poor taste in my mouth — I know my dopey $20 isn’t much, but it was the gesture; and it’s disappointing it isn’t paid in kind.

Tim

(I'm writing here in addition to a few catch-all email addresses at Vercel, in case there's some technical issue preventing our correspondence via email)

### Steps to Reproduce

See case 00142571

**Comment by swarnava** [comment, marked as answer]
Hi Tim,

Thank you for your patience. We just responded to you via support email. You can also check our Support SLA here: https://vercel.com/legal/support-terms

Thank you!
